diff -ruN a/src/utils/gcal-utils.c b/src/utils/gcal-utils.c
--- a/src/utils/gcal-utils.c	2024-09-16 15:28:53.000000000 +0200
+++ b/src/utils/gcal-utils.c	2024-10-27 17:35:14.414735546 +0100
@@ -339,52 +339,7 @@
 gint
 get_first_weekday (void)
 {
-  int week_start;
-
-#ifdef HAVE__NL_TIME_FIRST_WEEKDAY
-
-  union { unsigned int word; char *string; } langinfo;
-  gint week_1stday = 0;
-  gint first_weekday = 1;
-  guint week_origin;
-
-  langinfo.string = nl_langinfo (_NL_TIME_FIRST_WEEKDAY);
-  first_weekday = langinfo.string[0];
-  langinfo.string = nl_langinfo (_NL_TIME_WEEK_1STDAY);
-  week_origin = langinfo.word;
-  if (week_origin == 19971130) /* Sunday */
-    week_1stday = 0;
-  else if (week_origin == 19971201) /* Monday */
-    week_1stday = 1;
-  else
-    g_warning ("Unknown value of _NL_TIME_WEEK_1STDAY.\n");
-
-  week_start = (week_1stday + first_weekday - 1) % 7;
-
-#else
-
-  gchar *gtk_week_start;
-
-
-  /* Use a define to hide the string from xgettext */
-# define GTK_WEEK_START "calendar:week_start:0"
-  gtk_week_start = dgettext ("gtk30", GTK_WEEK_START);
-
-  if (strncmp (gtk_week_start, "calendar:week_start:", 20) == 0)
-    week_start = *(gtk_week_start + 20) - '0';
-  else
-    week_start = -1;
-
-  if (week_start < 0 || week_start > 6)
-    {
-      g_warning ("Whoever translated calendar:week_start:0 for GTK+ "
-                 "did so wrongly.\n");
-      week_start = 0;
-    }
-
-#endif
-
-  return week_start;
+  return 1;
 }
 
 /**
